doc/manual/src/release-notes/rl-next.md

```diff
@@ -12,6 +12,8 @@
 
 - Introduce a new built-in function [`builtins.convertHash`](@docroot@/language/builtins.md#builtins-convertHash).
 
+- `nix-env --query` now respects `--drv-path` with `--json`.
```

Comment:
I'm not sure what this is supposed to mean, could you please expand? The PR description provides no context.

Is this also documented in the manual?

Reply:
> I'm not sure what this is supposed to mean, could you please expand? The PR description provides no context.
> 
> Is this also documented in the manual?

It just means that drv-path works with --json

Reply:
```suggestion
- Fixed a bug where `nix-env --query` ignored `--drv-path` when `--json` was set.
```